packages/students/src/components/ProfilePage/StudentProfileEdit.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+
+import { getActiveBatches } from '../../services/batch';
+import { LoginHeader, LoginFooter } from '../../../../shared-components/LoginComponents';
+import { userProfilePropType } from './userProfilePropType';
+import { uploadFile } from '../../services/firebase';
+
+export class StudentProfileEditComponent extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      ...props.userData,
+      loading: true,
+    };
+
+    this.cityWiseBatches = {};
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Instead of binding, you can directly use arrow functions.
`this.handleSubmit  = () => {}`